### DISCUSSION TITLE
Supabase realtime client for Go

### DISCUSSION THREAD

**Discussion by snassr** [supabase/supabase#5889] [Show and tell]
I started work on a [realtime-go client](https://github.com/overseedio/realtime-go). I'm trying to get to start a discussion on how to move it to the supabase-community.

It is working and should closely align with the api of realtime-js. At the moment the only thing i have on the todo list is tests. 

Otherwise looking for feedback and usage!

Please let me know if this is duplicate work or if you find any issues, originally i needed to stream using Go from supabase but found no solution out there.

Thanks

**Comment by kiwicopple** [comment]
Hey @snassr ! I see you reached out on Discord - let's chat there. Thanks for the contribution 🎉

**Comment by donvito** [comment]
Hi guys, any updates on this? I tried @snassr but I am not able to get a handshake. I posted it as an issue in the repo. https://github.com/overseedio/realtime-go/issues/6

**Comment by flamxiOfWayland** [comment]
do we have any update on this ?
